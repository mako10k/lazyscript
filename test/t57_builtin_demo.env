LAZYSCRIPT_BUILTIN_PATH="$ROOT/src/plugins/.libs"
