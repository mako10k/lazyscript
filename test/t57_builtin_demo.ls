!{
  ~mod <- ((~prelude .builtin) "demo");
  ~~println (~to_str (((~mod .hello))));
  ~~println (~to_str (((~mod .add1) 41)));
};
