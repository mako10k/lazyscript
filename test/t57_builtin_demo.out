~builtin:demo#hello{-builtin/0-}
42
